Management/src/ActuatorApi/OpenTelemetryExtensions.cs

```diff
@@ -0,0 +1,53 @@
+using OpenTelemetry;
+using OpenTelemetry.Context.Propagation;
+using OpenTelemetry.Instrumentation.AspNetCore;
+using OpenTelemetry.Metrics;
+using OpenTelemetry.Resources;
+using OpenTelemetry.Trace;
+using Steeltoe.Common;
+using B3Propagator = OpenTelemetry.Extensions.Propagators.B3Propagator;
+
+namespace Steeltoe.Samples.ActuatorApi;
+
+internal static class OpenTelemetryExtensions
+{
+    public static void ConfigureOpenTelemetry(this IServiceCollection services, IConfiguration configuration)
+    {
+        services.AddOpenTelemetry().WithMetrics(metrics =>
+        {
+            metrics.AddAspNetCoreInstrumentation().AddHttpClientInstrumentation().AddRuntimeInstrumentation();
+        }).WithTracing(tracing =>
```

Comment:
The parameter was renamed from `tracerProviderBuilder` to `tracing` here, but the code below still uses the original name. While I prefer the longer name for clarity, such a change should be applied consistently.

Reply:
Fair point, but these names now match what comes in the Aspire ServiceDefaults templates. I do see that I missed renaming the builder in `ConfigureOpenTelemetryTracerProvider` though, so I can fix that

Reply:
I suspected so, but that doesn't mean we need to do the same. It's a slippery slope to shorten everything.

Reply:
ok